quacc/recipes/qchem/core.py

```diff
@@ -0,0 +1,103 @@
+"""
+Core recipes for the Q-Chem
+"""
+from __future__ import annotations
+
+from copy import deepcopy
+from typing import Literal
+
+from ase.atoms import Atoms
+from monty.dev import requires
+
+from quacc.schemas.ase import (
+    summarize_opt_run,
+    summarize_run,
+    summarize_thermo_run,
+    summarize_vib_run,
+)
+from quacc.util.calc import run_ase_opt, run_ase_vib, run_calc
+from quacc.util.thermo import ideal_gas
+from quacc.calculators.qchem import QChem
+
+def static_job(
```

Comment:
Before all jobs you'd consider to be a unique Slurm job, please add a `@ct.electron` dectorator above the function name (with `import covalent as ct` at the top). This will allow users to use Covalent as their manager if they wish while still also supporting Jobflow+FireWorks.

Reply:
fixed